### DISCUSSION TITLE
Current Issues/Concerns after Developer Meeting

### DISCUSSION THREAD

**Discussion by thabaum** [ADefWebserver/ADefWebserver.Module.HtmlTextV2#3] [General]
I would like to get a list of concerns going forward as I dive into this over the next few days.  I said I would help here and I know it is a big job/task to donate to the community so this is greatly appreciated I believe by many.

1. Security
2. Maintenance
3. Integration (Replacing current HTML editor seamlessly)
4. Features enabled/disabled
5. Server/Client WebAssembly compatibility
6. Interactive/Static compatibility

If I recall these are the main talking points discussed and we need to review while testing and reviewing.

**Comment by ADefWebserver** [comment]
1. **Security** - Ummmm
2. **Maintenance** - Ummmm
3. **Integration (Replacing current HTML editor seamlessly)** - Ok on this one I feel this module should read and write to the exact same tables at the default Quill module. So if you switch to this module everything just works :)
4. **Features enabled/disabled** - In Settings...
5. **Server/Client WebAssembly compatibility** - Unknown! May require additinal work
6. **Interactive/Static compatibility** - See above...

**Reply by thabaum** [reply]
I can surely help with 5 and 6 for now.

1. Security issue... do we need to grab the latest code related to HTML component from Radzen to update this area?  This workflow is what we would need to rely on to help maintain point number 2.

https://github.com/radzenhq/radzen-blazor/tree/master/Radzen.Blazor

**Reply by thabaum** [reply]
Link to current HTML editor issues on github for review: https://github.com/radzenhq/radzen-blazor/issues?q=is%3Aissue+is%3Aopen+HTML

I do not see any relating to security, but of coarse those are generally done discretely.

Trying to review any problems going forward.  Seems like this code is very solid from the actual current Radzen repository.

**Reply by thabaum** [reply]
One last thing, I like the syntax feature of this product... I would like to use it for a CSS styling feature for Site/Theme/Page/Module I may submit as an enhancement once this can be completed and merged if accepted.  I brought up this discussion at the end of the meeting.  I feel if we are trying to make it so users of CMS systems like DNN and WordPress can move to Oqtane, we need to cover the basics like CSS styles and content editing.

I see this as a fundamental cornerstone for Oqtane going forward personally.  I like the idea and feel both this and the HTML current module need a lot of love in the area of fixing/maintaining.

Busy today but I will get working on this tomorrow.  I am just trying to fill myself with all the details needed.  If nothing else I will create a module for personal use from it ;)

**Reply by ADefWebserver** [reply]
> 1. Security issue... do we need to grab the latest code related to HTML component from Radzen to update this area?

I say no because it-is-so-much-code (sigh). Integrating their latest with our current could take days.

**Reply by ADefWebserver** [reply]
> One last thing, I like the syntax feature of this product... I would like to use it for a CSS styling feature for Site/Theme/Page/Module I may submit as an enhancement once this can be completed and merged if accepted. I brought up this discussion at the end of the meeting. I feel if we are trying to make it so users of CMS systems like DNN and WordPress can move to Oqtane, we need to cover the basics like CSS styles and content editing.
> 
> I see this as a fundamental cornerstone for Oqtane going forward personally. I like the idea and feel both this and the HTML current module need a lot of love in the area of fixing/maintaining.
> 
> Busy today but I will get working on this tomorrow. I am just trying to fill myself with all the details needed. If nothing else I will create a module for personal use from it ;)

I agree with you 👍🏽

So where we are now is I want one more week to try to get the basic functionality working. Then we discuss what to do next :)
